public/js/components/AtomEdit/CustomEditors/CTAEditor.js

```diff
@@ -18,19 +18,19 @@ export class CTAEditor extends React.Component {
     return (
       <div className="editor editor-cta">
         <ManagedForm data={this.props.atom} updateData={this.props.onUpdate}>
-          <ManagedField fieldLocation="data.url" name="Link Url" isRequired={true}>
+          <ManagedField fieldLocation="data.url" name="Link Url" fieldId="url" isRequired={true}>
```

Comment:
Does this fieldId need to be provided? Can we get a unique if from the react component (might not be human readable)